xcp_d/utils/hcp2fmriprep.py

```diff
@@ -113,30 +143,61 @@ def hcpfmriprepx(hcp_dir, out_dir, subid):
 
     for i, j in zip(hcpfiles, newanatfiles):
         copyfileobj_example(i, j)
-
+    print("finished converting anat files")
     # get the task files
-    taskx = glob.glob(hcp_dir + "/" + subid + "/MNINonLinear/Results/*")
+
+    taskx = glob.glob(hcp_dir + "/Results/*")
     tasklistx = []
+    print(hcp_dir)
     for j in taskx:
         if j.endswith("RL") or j.endswith("LR"):
             tasklistx.append(j)
 
-    csf_mask = pkgrf("xcp_d", "masks/csf.nii.gz")
-    wm_mask = pkgrf("xcp_d", "masks/wm.nii.gz")
-
+    csf_mask = pkgrf("xcp_d", "/data/masks/csf.nii.gz")
+    wm_mask = pkgrf("xcp_d", "/data/masks/wm.nii.gz")
     for k in tasklistx:
         idx = os.path.basename(k).split("_")
         filenamex = os.path.basename(k)
 
         # create confound regressors
 
         mvreg = pd.read_csv(k + "/Movement_Regressors.txt", header=None, delimiter=r"\s+")
-        mvreg = mvreg.iloc[:, 0:6]
-        mvreg.columns = ["trans_x", "trans_y", "trans_z", "rot_x", "rot_y", "rot_z"]
+        mvreg = mvreg.iloc[:, :]
+        mvreg.columns = [
+            "trans_x",
+            "trans_y",
+            "trans_z",
+            "rot_x",
+            "rot_y",
+            "rot_z",
+            "trans_x_derivative1",
+            "trans_y_derivative1",
+            "trans_z_derivative1",
+            "rot_x_derivative1",
+            "rot_y_derivative1",
+            "rot_z_derivative1",
+        ]
         # convert rot to rad
         mvreg["rot_x"] = mvreg["rot_x"] * np.pi / 180
         mvreg["rot_y"] = mvreg["rot_y"] * np.pi / 180
         mvreg["rot_z"] = mvreg["rot_z"] * np.pi / 180
+        mvreg["rot_x_derivative1"] = mvreg["rot_x_derivative1"] * np.pi / 180
+        mvreg["rot_y_derivative1"] = mvreg["rot_y_derivative1"] * np.pi / 180
+        mvreg["rot_z_derivative1"] = mvreg["rot_z_derivative1"] * np.pi / 180
+        mvreg["trans_x_power2"] = mvreg["trans_x"] ** 2
+
+        # get derivatives and powers
```

Comment:
```suggestion

        # get derivatives and powers
        mvreg["trans_x_power2"] = mvreg["trans_x"] ** 2
```
Just moving it down to be with similar assignments.